### ISSUE TITLE
TestTensorExprFuser.test_unused is failing in release/1.6 branch.

### ISSUE THREAD

**Issue by choidongyeon** [pytorch/pytorch#41358]
## 🐛 Bug

I am on the ```release/1.6``` branch. I have not made any changes to the code and TestTensorExprFuser.test_unused is failing:

```
FAIL: test_unused (__main__.TestTensorExprFuser)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "test/test_tensorexpr.py", line 1249, in test_unused
    assert cx.elapsed_value() == 1
AssertionError
```

## To Reproduce

```
FAILED (failures=1, skipped=1)
python test/test_tensorexpr.py TestTensorExprFuser.test_unused
```

## Expected behavior

Test should pass.

## Environment

```
Collecting environment information...
PyTorch version: N/A
Is debug build: N/A
CUDA used to build PyTorch: N/A

OS: Amazon Linux 2
GCC version: (GCC) 7.3.1 20180712 (Red Hat 7.3.1-6)
CMake version: version 3.13.3

Python version: 3.7
Is CUDA available: N/A
CUDA runtime version: 10.0.130
GPU models and configuration: 
GPU 0: Tesla V100-SXM2-16GB
GPU 1: Tesla V100-SXM2-16GB
GPU 2: Tesla V100-SXM2-16GB
GPU 3: Tesla V100-SXM2-16GB

Nvidia driver version: 440.33.01
cuDNN version: Probably one of the following:
/usr/local/cuda-10.1/targets/x86_64-linux/lib/libcudnn.so.7.6.5
/usr/local/cuda-10.2/targets/x86_64-linux/lib/libcudnn.so.7.6.5

Versions of relevant libraries:
[pip3] numpy==1.18.5
[conda] blas                      1.0                         mkl  
[conda] mkl                       2020.0                      166  
[conda] mkl-service               2.3.0            py37he904b0f_0  
[conda] mkl_fft                   1.0.15           py37ha843d7b_0  
[conda] mkl_random                1.1.0            py37hd6b4f25_0  
[conda] numpy                     1.18.1           py37h4f9e942_0  
[conda] numpy-base                1.18.1           py37hde5b4d6_1  
[conda] numpydoc                  0.9.2                      py_0  
[conda] torch                     1.6.0a0+cefb9e0          pypi_0    pypi

```

cc @suo @gmagogsfm

**Comment by Flamefire** [comment]
Happens for me too.

PyTorch 1.6-rc2
CUDA 10.1
GPU: Tesla K80

**Comment by ezyang** [comment]
cc @malfet @seemethere

**Comment by malfet** [comment]
This test is disabled in 1.6 and was re-enabled in master only today, see https://github.com/pytorch/pytorch/commit/3971777ebbad5b66142c99568dcf8efbe1c73891
cc @Krovatkin

**Comment by Flamefire** [comment]
> was re-enabled in master only today

I've seen that it wasn't just reenabled but an assertion has been changed to assert a result of 0 instead of 1 so it seems the test was wrong in the first place and fixed by 3971777. I can confirm that applying this change works on the 1.6 branch (rc3 tag). So IMO it would be worth applying it to the 1.6 release

**Comment by gchanan** [comment]
We aren't planning to do a 1.6.1 release at this time, so closing for now.
